boot/build.go

```diff
@@ -324,3 +328,14 @@ func friendlySize(size float64) string {
 
 	return fmt.Sprintf("%0.1f %s", size, unit)
 }
+
+func FindSpringCloudBindingsJars(jars []libjvm.MavenJAR) *libjvm.MavenJAR {
+	var springCloudBindingsJar *libjvm.MavenJAR
+	springCloudBindingsName := "spring-cloud-bindings"
+	for _, lib := range jars {
+		if lib.Name == springCloudBindingsName {
+			springCloudBindingsJar = &lib
```

Comment:
There is a bug here. The version output in my log is not correct, I was getting '9.0.69' which is the Tomcat lib versions. This is because you are assigning springCloudBindingsJar the address of 'lib' which changes on each loop, thereby it always ends up with the last value after the first time the 'if' statement returns true.

If you **do** keep the version extraction logic and thus keep returning the reference, you could break/return early here, there's no need to continue the loop once you find a match anyway. You could also simply pass the value instead of using pointers.

Reply:
I would add a follow-up point here, in that this bug slipped through the unit tests. The test coverage looks good on the surface, but something must be off since this didn't get caught. My suggestion would be to modify the unit tests to catch the bug that @pivotal-david-osullivan pointed out and then fix it.

Reply:
I could reproduce the issue in the test, after appending another `libjvm.MavenJAR` into `mavenJars`

I missed that bug because in my test the `springCloudBindingsJar` was the last jar I added.

That said, I'll probably go down the route suggested by @pivotal-david-osullivan , ie not returning the version anymore, and stopping the loop after the first hit, returning a boolean